resources/kubernetes/image-build/build_latest.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# This will build Docker images for all the latest WSO2 products to use in testgrid
+set -o xtrace
+
+LOG_FILE="$(date +%F).log"
+
+# Git repo URLs
+APIM_GIT_REPO_URL_260="https://github.com/NishikaDeSilva/docker-apim-support/archive/v2.6.0.1.zip"
+EI_GIT_REPO_URL_640="https://github.com/NishikaDeSilva/docker-ei-support/archive/v6.4.0.zip"
+IS_GIT_REPO_URL_570="https://github.com/NishikaDeSilva/docker-is-support/archive/v5.7.0.zip"
+
+# name of the unzipped directory
+APIM_GIT_REPO_NAME_260="docker-apim-support-2.6.0.1"
+EI_GIT_REPO_NAME_640="docker-ei-support-6.4.0"
+IS_GIT_REPO_NAME_570="docker-is-support-5.7.0"
+
+echo "----------------------------------------Building new images with latest updates--------------------------------------------------"
+
+echo "----------------------------------------Building wso2am-2.6.0---------------------------------------------------------"
```

Comment:
Can we iterate the three products as an array via a for-loop? That way, we can avoid code-duplication, and it'll also allow for extending this to support other product versions.

Reply:
sounds good. Ill work on it